app/jobs/reports/base_report.rb

```diff
@@ -47,13 +47,28 @@ def transaction_with_timeout(rails_env = Rails.env)
     end
 
     def save_report(report_name, body, extension:)
+      write_json_report_to_analytics(report_name, body, extension)
       if !IdentityConfig.store.s3_reports_enabled
         logger.info('Not uploading report to S3, s3_reports_enabled is false')
         return body
       end
       upload_file_to_s3_timestamped_and_latest(report_name, body, extension)
     end
 
+    def write_json_report_to_analytics(report_name, body, extension)
+      return unless extension == 'json'
+      build_analytics.track_event(
+        Analytics::REPORT_RESULTS,
+        user_id: '',
```

Comment:
```suggestion
```
I don't think is `user_id` required, what if we not just leave it out? We can pass an instance of `AnonymousUser` in `build_analytics` below so that `user.uuid` does not blow up

Reply:
Technically it's not an anonymous user but I don't have strong feeling either way so I'll just push this.